pages/common/aria2.md

```diff
@@ -1,33 +1,7 @@
 # aria2
 
-> A lightweight multi-protocol & multi-source command-line download utility.
-> Supports HTTP, HTTPS, FTP, SFTP, BitTorrent and Metalink.
-> More information: <https://aria2.github.io/>.
+> This command is an alias of `aria2c`.
 
-- Download a web resource:
+- View documentation for the original command:
```

Comment:
```suggestion
- View documentation for the correct command:
```
opinions?

Reply:
Hmm, this is tricky.  We can change the wording in alias pages depending on the nature of the alias page itself.  Perhaps `new`?

Reply:
Wont always be new, in 10 years time it will be _pretty old_

Reply:
Then let's keep it as `original`, how about.

Reply:
That makes it sound like aria2 was a new version, `updated` maybe?

Reply:
Sounds good to me.
```suggestion
- View documentation for the updated command:
```